src/app/components/challenge/challengephases/phasecard/phasecard.component.ts

```diff
@@ -81,10 +81,17 @@ export class PhasecardComponent implements OnInit {
     SELF.apiCall = (params) => {
       const FORM_DATA: FormData = new FormData();
       for (const key in params) {
-        if (params[key]) {
-          FORM_DATA.append(key, params[key]);
+       if (this.challengeService.isValidSubmission(params)) {
```

Comment:
Can we change the name to make more sense here something like `isMaxSubmissionsCountValid`.